config/storage/middleware/stores/patching.json

```diff
@@ -31,6 +32,32 @@
         }
       }
     },
+    {
+      "@id": "urn:solid-server:default:PatchHandler_RDFStore",
+      "@type": "WaterfallHandler",
+      "handlers": [
+        { "@id": "urn:solid-server:default:PatchHandler_ImmutableCheck"},
+        { "@id": "urn:solid-server:default:PatchHandler_RDF" }
+      ]
+    },
+    {
+      "comment": "Patches metadata resources. Prevents specific triple patterns from being updated.",
+      "@id": "urn:solid-server:default:PatchHandler_ImmutableCheck",
+      "@type": "ImmutableMetadataPatcher",
+      "patcher": { "@id": "urn:solid-server:default:PatchHandler_RDF" },
+      "metadataStrategy": { "@id": "urn:solid-server:default:MetadataStrategy" },
+      "immutablePatterns": [
```

Comment:
Some other patterns that should be in here: 

The predicate used to indicate acl resources: https://github.com/CommunitySolidServer/CommunitySolidServer/blob/77eeae9c9e89db2d4c87e9a5c1b28fdee6608d5d/config/ldp/metadata-writer/writers/link-rel.json#L14

Patterns related to file size: https://github.com/CommunitySolidServer/CommunitySolidServer/blob/ff80079000a0f308f31b83cfc52e90421f20dadc/src/storage/accessors/FileDataAccessor.ts#L322-L327

Patterns indicating the modified date: https://github.com/CommunitySolidServer/CommunitySolidServer/blob/b3da9c9fcfb642e8a8a6d537b2e69cbd74ee8a88/src/util/ResourceUtil.ts#L35

The content-type: https://github.com/CommunitySolidServer/CommunitySolidServer/blob/fbbccb0cf1f7980c008ecc06cabccb8c08b48b5f/src/util/Vocabularies.ts#L184

There might be some more I'm forgetting at the moment. Might not be bad to have a look at what exactly is contained in a `RepresentationMetadata` object during input and output.